src/main/resources/manifest.yml

```diff
@@ -1,3 +1,3 @@
-manifestVersion: 1.0.0-alpha
-productName: @serviceName@
-productVersion: @serviceVersion@
+manifest-version: 1.0
+product-name: @serviceName@
+product-version: @serviceVersion@
```

Comment:
`product-group` and `product-type` are [required](https://github.com/palantir/sls-spec/blob/master/manifest.md) in Manifest 1.0, we should fill them in here too. The group we can get from the gradle project; for the type we might need another field on the DistributionExtension